components/resources/tooltip_ui_resources.grdp

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
copyright here and other files.

Reply:
I haven't seen copyright header in any of the grdp files. Which other files are you referring to here?